.github/workflows/owasp_zap.yml

```diff
@@ -24,6 +24,8 @@ jobs:
           path: ~/.gradle/caches
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
           restore-keys: ${{ runner.os }}-gradle
+      - name: Verify Docker Compose installation
```

Comment:
This is why I've generally just started always using test-containers. I think it does have some support for docker-compose.

Reply:
Dunno why it stopped working all of a sudden!